lightning/src/ln/channel.rs

```diff
@@ -4195,6 +4239,22 @@ impl<Signer: Sign> Channel<Signer> {
 		self.short_channel_id
 	}
 
+	/// Allowed in any state (including after shutdown)
+	pub fn latest_inbound_scid_alias(&self) -> Option<u64> {
+		self.latest_inbound_scid_alias
+	}
+
+	/// Allowed in any state (including after shutdown)
+	pub fn outbound_scid_alias(&self) -> u64 {
+		self.outbound_scid_alias
+	}
+	/// Only allowed immediately after deserialization if get_outbound_scid_alias returns 0,
+	/// indicating we were written by LDK prior to 0.0.106 which did not set outbound SCID aliases.
+	pub fn set_outbound_scid_alias(&mut self, outbound_scid_alias: u64) {
```

Comment:
I think you can precise the requirements of the given `outbound_scid_alias` if this channel utility is re-used in the future.

Reply:
I have no idea what you mean here?

Reply:
What's the requirements about the `u64` ? Like at least to not give the _real_ channel SCID as an argument.

Reply:
Recall `channel.rs` is internal-only, so this isn't for external consumption, and is pretty clear given it says `outbound_scid_alias` in the title :)